mvp-0/values-observer.yaml

```diff
@@ -12,14 +26,22 @@ thanos:
       type: NodePort
       nodePorts:
         remote: 30291
+  compactor:
```

Comment:
Maybe compactor should be moved to the `thanos-storage.yaml` as it is only relevant for object storage

Also another question. Does it make sense to also enable bucketweb component there?

Reply:
Maybe yes, if you made separate values for that,. But I do not think this is good when you have doubled values e.g. `thanosRuler` in both in `observer-values.yaml` and also in `thanos-storage.yaml`.

For me no, cause it just a MVP.

Reply:
We have separate values for that. The user has the option to install mvp-0 without object storage, e.g. for local deployment with KinD.

Reply:
There is no doubling of values, every value is defined only once

Reply:
> For me no, cause it just a MVP.

Ok, for mvp-0 we do not need bucketweb, storegateway component UI is enough.

Reply:
> We have separate values for that. The user has the option to install mvp-0 without object storage, e.g. for local deployment with KinD.

Ok, compactor is moved to the `thanos-storage.yaml`